aspnetcore/testing/razor-pages-testing.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Razor Pages unit and integration testing in ASP.NET Core
+author: guardrex
+description: Learn how to create unit and integration tests for Razor Pages apps.
+ms.author: riande
+manager: wpickett
+ms.date: 11/20/2017
+ms.topic: article
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: testing/razor-pages-testing
+---
```

Comment:
Please add the following metadata:
```
ms.custom: mvc
```

Reply:
@scottaddie This topic is for the "testing" area and uses an RP app, so it seems strange to label it "mvc" at all. Why is it added? ... and are all topics getting this?

Reply:
Sorry I wasn't clearer.

Reply:
No worries. It led to a good laugh when I asked why this non-MVC topic was tagged "mvc." lol :smile: